package.json

```diff
@@ -65,77 +65,77 @@
   "author": "nicolodavis@gmail.com",
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.5.4",
-    "@babel/node": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "^7.5.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
-    "@babel/preset-env": "^7.5.4",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.3.3",
-    "@firebase/app": "^0.4.26",
-    "@testing-library/jest-dom": "^4.1.2",
-    "@testing-library/svelte": "^1.9.0",
-    "@types/jest": "^24.0.0",
-    "ajv": "^6.6.2",
-    "babel-core": "^7.0.0-bridge.0",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.6",
+    "@babel/node": "^7.8.4",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.8.3",
+    "@babel/preset-env": "^7.8.6",
+    "@babel/preset-react": "^7.8.3",
+    "@babel/preset-typescript": "^7.8.3",
+    "@firebase/app": "^0.5.5",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/svelte": "^1.11.0",
+    "@types/jest": "^24.9.1",
+    "@types/node": "^12.0.0",
+    "ajv": "^6.12.0",
     "babel-eslint": "^8.0.0",
     "babel-loader": "^8.0.5",
     "babel-plugin-module-resolver": "^3.0.0",
     "benchmark": "^2.1.4",
     "bundlewatch": "^0.2.5",
     "concurrently": "^4.1.0",
-    "coveralls": "^3.0.0",
+    "coveralls": "^3.0.9",
     "cross-env": "^5.1.1",
-    "css-loader": "^3.1.0",
-    "docsify-cli": "^4.3.0",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
+    "css-loader": "^3.4.2",
+    "docsify-cli": "^4.4.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^5.6.0",
     "eslint-config-prettier": "^3.1.0",
     "eslint-plugin-jest": "^21.22.1",
     "eslint-plugin-prettier": "^2.6.2",
-    "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-unicorn": "^6.0.1",
-    "firebase": "^7.5.2",
-    "firebase-admin": "^8.5.0",
-    "firebase-mock": "^2.2.10",
-    "graceful-fs": "https://github.com/mekwall/node-graceful-fs.git#patch-1",
```

Comment:
There are more changes to this file than are necessary to fix the problem at hand.  As such, I'm not comfortable approving this PR unless it is broken into smaller simpler changes.

Would merely removing `graceful-fs` be sufficient?  Is there a need to upgrade everything else?  (not opposed to upgrading them in a separate PR of course).

Reply:
Oh, looks like I ran an npm upgrade too much. Yeah, I will break it down.
Just removing graceful-fs should be good enough.